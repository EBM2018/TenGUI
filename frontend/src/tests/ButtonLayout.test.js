import renderer from 'react-test-renderer';
import React from 'react';
import ButtonLayout from '../react/Fishtank/ButtonLayout/ButtonLayout';

test('derp', () => {
  expect('True').toBeTruthy;
});
test('renders correctly', () => {
  const component = renderer
    .create(<ButtonLayout />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
