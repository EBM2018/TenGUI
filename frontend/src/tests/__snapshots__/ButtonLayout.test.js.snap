// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="bg-color"
>
  <div
    className="columns add-margin "
  >
    <div
      className="column is-3"
    >
      <div
        className="field"
      >
        <p
          className="control has-icons-left has-icons-right"
        >
          <input
            className="input"
            onKeyUp={[Function]}
            placeholder="Posez votre question"
            type="text"
          />
          <span
            className="icon is-small is-left"
          >
            <i
              className="fas fa-hand-point-up"
            />
          </span>
        </p>
      </div>
    </div>
    <div
      className="column is-3 has-text-left dropdown"
    >
      <p
        className="control has-icons-left has-icons-right"
      >
        <button
          className="button is-info is-inverted is-fullwidth"
          name="button"
          type="button"
        >
          Changement de rythme
        </button>
        <span
          className="icon is-small is-left"
        >
          <i
            className="fas fa-question-circle"
          />
        </span>
      </p>
      <div
        className="dropdown-content"
      >
        <p
          className="control has-icons-left has-icons-right"
        >
          <button
            className="button is-info is-inverted  is-fullwidth buttonList"
            onClick={[Function]}
            type="button"
          >
            Demander d aller plus vite
          </button>
          <span
            className="icon is-small is-left"
          >
            <i
              className="fas fa-question-circle"
            />
          </span>
        </p>
        <p
          className="control has-icons-left has-icons-right"
        >
          <button
            className="button is-info is-inverted  is-fullwidth buttonList"
            onClick={[Function]}
            type="button"
          >
            Demander d aller moins vite
          </button>
          <span
            className="icon is-small is-left"
          >
            <i
              className="fas fa-question-circle"
            />
          </span>
        </p>
        <p
          className="control has-icons-left has-icons-right"
        >
          <button
            className="button is-info is-inverted  is-fullwidth buttonList"
            onClick={[Function]}
            type="button"
          >
            Je ne comprends plus rien
          </button>
          <span
            className="icon is-small is-left"
          >
            <i
              className="fas fa-question-circle"
            />
          </span>
        </p>
        <p
          className="control has-icons-left has-icons-right"
        >
          <button
            className="button is-info is-inverted  is-fullwidth buttonList"
            onClick={[Function]}
            type="button"
          >
            Demander une pause
          </button>
          <span
            className="icon is-small is-left"
          >
            <i
              className="fas fa-question-circle"
            />
          </span>
        </p>
      </div>
    </div>
    <div
      className="column is-3 dropdown"
    >
      <p
        className="control has-icons-left has-icons-right"
      >
        <button
          className="button is-info is-inverted  is-fullwidth"
          name="button"
          type="button"
        >
          Précision
        </button>
        <span
          className="icon is-small is-left"
        >
          <i
            className="fas fa-plus-circle"
          />
        </span>
      </p>
      <div
        className="dropdown-content"
      >
        <p
          className="control has-icons-left has-icons-right"
        >
          <button
            className="button is-info is-inverted  is-fullwidth buttonList"
            onClick={[Function]}
            type="button"
          >
            Demander de réexpliquer
          </button>
          <span
            className="icon is-small is-left"
          >
            <i
              className="fas fa-question-circle"
            />
          </span>
        </p>
        <p
          className="control has-icons-left has-icons-right"
        >
          <button
            className="button is-info is-inverted  is-fullwidth buttonList"
            onClick={[Function]}
            type="button"
          >
            Demander de détailler
          </button>
          <span
            className="icon is-small is-left"
          >
            <i
              className="fas fa-question-circle"
            />
          </span>
        </p>
        <p
          className="control has-icons-left has-icons-right"
        >
          <button
            className="button is-info is-inverted  is-fullwidth buttonList"
            onClick={[Function]}
            type="button"
          >
            Demander un exemple
          </button>
          <span
            className="icon is-small is-left"
          >
            <i
              className="fas fa-question-circle"
            />
          </span>
        </p>
        <p
          className="control has-icons-left has-icons-right"
        >
          <button
            className="button is-info is-inverted  is-fullwidth buttonList"
            onClick={[Function]}
            type="button"
          >
            Demander une anecdote
          </button>
          <span
            className="icon is-small is-left"
          >
            <i
              className="fas fa-question-circle"
            />
          </span>
        </p>
        <p
          className="control has-icons-left has-icons-right"
        >
          <button
            className="button is-info is-inverted  is-fullwidth buttonList"
            onClick={[Function]}
            type="button"
          >
            Demander une référence
          </button>
          <span
            className="icon is-small is-left"
          >
            <i
              className="fas fa-question-circle"
            />
          </span>
        </p>
        <p
          className="control has-icons-left has-icons-right"
        >
          <button
            className="button is-info is-inverted  is-fullwidth buttonList"
            onClick={[Function]}
            type="button"
          >
            Demander un exercice
          </button>
          <span
            className="icon is-small is-left"
          >
            <i
              className="fas fa-question-circle"
            />
          </span>
        </p>
      </div>
    </div>
    <div
      className="column is-2"
    >
      <p
        className="control has-icons-left has-icons-right"
      >
        <button
          className="button is-danger is-fullwidth"
          onClick={[Function]}
          type="button"
        >
          Arrêt d'urgence
        </button>
        <span
          className="icon is-small is-left"
        >
          <i
            className="far fa-stop-circle"
          />
        </span>
      </p>
    </div>
    <div
      id="clean"
    />
  </div>
</div>
`;
